### PR TITLE
Poll isRunning from conferences minutely and notify users

### PR DESCRIPTION
**Pull request by TomlDev**
See issue

[Update .env](https://doku.netzint.de/doku.php?id=intern:technik:schulserver:edulution:guides:dev-env)

### CODE REVIEW

File: apps/api/src/conferences/conferences.service.ts [hunk 1]
```diff
@@ -17,23 +20,81 @@ import type GroupWithMembers from '@libs/groups/types/groupWithMembers';
...
+
+  private pollInterval = Number.parseInt(BBB_POLL_INTERVAL, 10) || 60000;
+
+  scheduleConferenceSync() {
+    const interval = setInterval(() => {
+      void this.syncRunningConferencesStatus();
+    }, this.pollInterval);
+
+    this.schedulerRegistry.addInterval(CONFERENCES_SYNC_INTERVAL, interval);
+  }
```
Comment (michidold) [inline apps/api/src/conferences/conferences.service.ts:63]: Is there a reason why this is not handled by the `@Interval` decorator as in scheduled groups fetch?

**Review by michidold — CHANGES_REQUESTED** [review]
One small question.

**Review by michidold — CHANGES_REQUESTED** [review]
I still have a issue were the running state is not updated or throws false errors:
[Bildschirmaufzeichnung vom 2024-11-19 11-15-56.webm](https://github.com/user-attachments/assets/bae0f93b-025f-48ba-baf9-ee93846c6204)

**Review by michidold — CHANGES_REQUESTED** [review]
Have a issue when starting the conference. See comment.

**Review by michidold — APPROVED** [review]
Looks good. Maybe we can optimize that in another task by opening the frame while its loading

File: apps/api/src/conferences/conferences.service.ts
[the remark this answers is not in the document]
Comment (michidold) [reply apps/api/src/conferences/conferences.service.ts:1]: Maybe we can optimize that in another task by opening the frame while its loading

**Comment by TomlDev** [comment]
Created a task:
https://github.com/edulution-io/edulution-ui/issues/288

**Pull request merged**